sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -142,6 +142,51 @@ public async Task StartRecognizeContentPopulatesFormPage(bool useStream)
             }
         }
 
+        [Test]
+        [TestCase(true)]
+        [TestCase(false, Ignore = "File has not been uploaded to GitHub yet.")]
+        public async Task StartRecognizeContentCanParseMultipagedForm(bool useStream)
+        {
+            var client = CreateInstrumentedFormRecognizerClient();
+            var filename = "multipage_invoice_noblank.pdf";
```

Comment:
curious why this filename is not in the `FormRecognizerTestEnvironment` class

Reply:
You have a point. I was using `CreatePath` and `CreateUri` to avoid making the `FormRecognizerTestEnvironment` messy, but I'll create a `MultipageFormPath` and `MultipageFormUri` for now. I really think we should revisit some points of the test infrastructure in the future (but definitely a low-priority issue).